library/yourls

```diff
@@ -4,17 +4,17 @@ Maintainers: YOURLS <yourls@yourls.org> (@YOURLS),
              Léo Colombaro <git@colombaro.fr> (@LeoColomb)
 GitRepo: https://github.com/YOURLS/docker-yourls.git
 
-Tags: 1.7.2-apache, 1.7-apache, 1-apache, apache, 1.7.2, 1.7, 1, latest
```

Comment:
Did you want to lose all these tag aliases, here and below? It looks like you just need a bump in [`generate-stackbrew-library.sh`](https://github.com/YOURLS/docker-yourls/blob/652a3971b92ab2e0bfa96ec78342f8757071dbf1/generate-stackbrew-library.sh#L5).

Reply:
Oops, good catch! 😇

Reply:
Fixed, thanks!